app/components/footer/talk_to_us_component.html.erb

```diff
@@ -18,13 +18,15 @@
               </div>
             </div>
             <p>Monday to Friday between <strong>8:30am and 5:30pm</strong></p>
+            <p>Chat will be unavailable from 1pm on 24 December until 29 December. It will also be closed on 3 January. Responses may be slower over the Christmas period.</p>
           </div>
           <div>
             <h3>Get a teacher training adviser</h3>
             <p>
               An experienced teaching professional can help if you're ready to get into
               teaching or if you're returning to teaching and qualified to teach maths,
               physics or languages.
+              <p>It may take slightly longer to be assigned an adviser between 24 December and 2 January.</p>
             </p>
             <a href="/tta-service" class="button">
```

Comment:
```suggestion
            <p>
              An experienced teaching professional can help if you're ready to get into
              teaching or if you're returning to teaching and qualified to teach maths,
              physics or languages.
            </p>
            
            <p>It may take slightly longer to be assigned an adviser between 24 December and 2 January.</p>
            
            <a href="/tta-service" class="button">
```

We'd usually try not to nest paragraph (`<p>` tags) within other paragraphs.